packages/form-js/CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ All notable changes to [form-js](https://github.com/bpmn-io/form-js) are documen
 
 ___Note:__ Yet to be released changes appear here._
 
+## 0.1.0
+
+_This release adds new fields. Older form-js viewers are incompatible with it._
```

Comment:
So we market this as a breaking change even though compatibility is given if you don't use any of the new form fields?

Reply:
We could be better in our communication here. Given our recent strategy it could be

> _This release adds new fields that are unsupported in older form-js viewers._

Reply:
Sounds good 👍🏻

Reply:
I went ahead and changed the wording: https://github.com/bpmn-io/form-js/commit/da87bcd0834d91b3326b0c71fdf649fcb76a99d3